### PR TITLE
Feature/cor 1705 correct timeseries agegroups

### PR DESCRIPTION
**Pull request by VWSCoronaDashboard29**
Remove underreported annotation from agegroups timeseries

### CODE REVIEW

**Review by VWSCoronaDashboard28 — COMMENTED** [review]
Something seems off in this PR. I'm seeing over 250 commits. What's going on?

File: packages/cms-v2/src/schemas/topical/theme-tile.ts
Comment (VWSCoronaDashboard30) [inline packages/cms-v2/src/schemas/topical/theme-tile.ts:1]: @VWSCoronaDashboard29 Why are we changing changing stuff to V2?

**Review by VWSCoronaDashboard30 — COMMENTED** [review]
Why are there 500 commits to the branch?

**Review by VWSCoronaDashboard28 — CHANGES_REQUESTED** [review]
Something is off given the number of commits, please recreate this PR with only the changes you have introduced.

**Pull request closed without merging**